src/content/reference/react/directives.md

```diff
@@ -1,10 +1,17 @@
 ---
 title: "Directives"
+canary: true
 ---
 
+<Canary>
+
+Directives are available in the [React Canary](https://react.dev/community/versioning-policy#canary-channel) release channel. They are needed only if you're [using React Server Components](/learn/start-a-new-react-project#bleeding-edge-react-frameworks) or building a library compatible with them.
```

Comment:
nit: the channel is called Canary, not React Canary. “release channel” would need to be linked before “React”